specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/Clusters.json

```diff
@@ -632,12 +632,6 @@
           "description": "Minimum CapacityReservation value in GB.",
           "type": "integer",
           "format": "int64"
-        },
-        "maxCapacity": {
```

Comment:
Please go through the Breaking Change Process outlined in the PR description to get approval for this change or show an existing approval for it

Reply:
@pilor  breaking change approval is on my last closed PR (opened new one to arrange commits)-> https://github.com/Azure/azure-rest-api-specs/pull/14598#partial-pull-merging